port/win/env_win.cc

```diff
@@ -2071,8 +2077,13 @@ class WinEnv : public Env {
   std::vector<ThreadPool> thread_pools_;
   mutable std::mutex mu_;
   std::vector<std::thread> threads_to_join_;
+  static FnGetSystemTimePreciseAsFileTime GetSystemTimePreciseAsFileTime_;
+  static bool GetSystemTimePreciseAsFileTimeInitialized_;
 };
 
+FnGetSystemTimePreciseAsFileTime WinEnv::GetSystemTimePreciseAsFileTime_ = NULL;
```

Comment:
You are right. I did not want to do lookup for every env, but there will be in most cases only one. Std call_once is also nice suggestion. I will make it as standard member, where no synchronization is needed.